src/main/java/com/example/TechItEasy/exceptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.TechItEasy.exceptions;
+
+import java.io.Serial;
+
+public class RecordNotFoundException extends RuntimeException {
+        @Serial
```

Comment:
Dit is geen SpringBoot annotatie. Deze "serialVersionUID" kan gebruikt worden tijdens serialisation/deserialization om te garanderen dat het ook werkelijk de juiste klasse is. Serialization is wanneer je objecten omzet naar iets anders, bijvoorbeeld bytes, voor opslag of het versturen over het netwerk. We doen daar hier niks mee en je mag dit ook direct vergeten.